.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ] You have mutiple .DS Store files. Please add them to gitignore to make your PR clean.

Reply:
Hello @arslanbisharat , I think this task is not required but I have added them as mentioned.